app/GRZMAK/GRZMAK_task2.c

```diff
@@ -0,0 +1,81 @@
+/**
+* Responsible: Grzegorz Makowski
+*
+*
+*History Log:
+* 2017.12.06 GRZMAK initial version
+*
+**/
+
+
+#include "GRZMAK_task2.h"
+
+
+
+eErr_t GRZMAK_setLampBit( eLamp_t bit, int* reg)
```

Comment:
Only one return statement per function is allowed `(MISRA, rule 14.7 : required) "A function shall have a single point of exit at the end of the function"`